apps/projects/app/components/Content/Filters.js

```diff
@@ -79,42 +96,39 @@ export default class Filters extends Component {
     if (Object.keys(filterAliases).length === 0) return null
 
     return (
-      <div style={{
-        marginLeft: '8px',
-        marginTop: '10px',
-        flexDirection: 'row',
-        display: 'flex',
-        flex: '1',
-        flexWrap: 'wrap'
-      }}>
-        {Object.keys(filterAliases).map(alias => {
-          const pathToDisableFilter = filterAliases[alias]
-          return (
-            <FilterTile
-              key={pathToDisableFilter.join('')}
-              text={alias}
-              disableFilter={() => this.props.disableFilter(pathToDisableFilter)}
-            />
-          )
-        })}
-        {Object.keys(filterAliases).length > 0 &&
-          <div
+      <Wrap>
+        <div>
+          {Object.keys(filterAliases).map(alias => {
+            const pathToDisableFilter = filterAliases[alias]
+            return (
+              <FilterTile
+                key={pathToDisableFilter.join('')}
+                text={alias}
+                disableFilter={() =>
+                  this.props.disableFilter(pathToDisableFilter)
+                }
+              />
+            )
+          })}
+        </div>
+        {Object.keys(filterAliases).length > 0 && (
+          <Button
+            mode="text"
+            size="small"
             onClick={this.props.disableAllFilters}
-            role='button'
```

Comment:
In general, I'm in favor of using `button` elements for buttons, rather than divs with `role="text"`. You may have noticed I left other such divs in place, though, in this PR. Why?

I tried to stay focused and only change things related to the styling issue at hand. This `div` wasn't vertically centered correctly. The button is.